kimchi-stubs/src/pasta_fp_plonk_index.rs

```diff
@@ -160,9 +161,7 @@ pub fn caml_pasta_fp_plonk_index_read(
     }
 
     // deserialize the index
-    let mut t = ProverIndex::<Vesta, OpeningProof<Vesta>>::deserialize(
-        &mut rmp_serde::Deserializer::new(r),
-    )?;
+    let mut t = ProverIndex::<55, Vesta, Srs>::deserialize(&mut rmp_serde::Deserializer::new(r))?;
```

Comment:
The reason this works out of the box is because the change from `Vec` to `array` in poseidon, only affects two static methods on the `KimchiCurve` trait.